recipe/meta.yaml

```
@@ -30,13 +30,23 @@ source:
   sha256: 908742d8f3c6fdd6d1d6316a7b919b5c506474f9551f491aa7335cb4f50bffbd  # [win]
 
 build:
-  number: 1
+  number: 2
   skip: true  # [osx]
 
 test:
+  requires:
+    - cmake  # [linux]
+  files:
+    - verify-version.cmake  # [linux]
   commands:
     - test -d $PREFIX/lib/cmake                      # [linux]
     - test -d $PREFIX/targets/{{ target_name }}/lib  # [linux]
+    - if not exist %LIBRARY_LIB%\{{ target_name }}\cmake exit 1    # [win]
+    - if not exist %LIBRARY_INC%\targets\{{ target_name }} exit 1  # [win]
+    # Extract the CCCL version from the cuda_cccl archive, and compare to the
+    # version of the cccl conda package.
+    - export CCCL_VERSION_FILE="$PREFIX/targets/{{ target_name }}/lib/cmake/cccl/cccl-config-version.cmake"  # [linux]
+    - '[[ "$(cmake -P verify-version.cmake 2>&1)" == "{{ cccl_version }}" ]] || exit 1'                      # [linux]
```

Something to consider: instead of passing `CCCL_VERSION_FILE` as an environment variable, you could pass it as a CMake variable with `-D`:

```suggestion
    - '[[ "$(cmake -DCCCL_VERSION_FILE="$PREFIX/targets/{{ target_name }}/lib/cmake/cccl/cccl-config-version.cmake" -P verify-version.cmake 2>&1)" == "{{ cccl_version }}" ]] || exit 1'                      # [linux]
```

---

My reason for using an env var was just that it split the test command across two lines more cleanly than one long line with an inline variable. Super minor. I’ll adapt according to your suggestion.